2_Controllers/test_controller.js

```diff
@@ -0,0 +1,11 @@
+const { fetchTopics } = require("../1_Models/test_model");
+const topics = require("../db/data/test-data/topics");
+
+const getTopics = (request, response, next) => {
+  fetchTopics().then((topics) => {
+    console.log("model is in controller");
+    response.status(200).json({ topics });
```

Comment:
Both .json() and .send() can do the work, but .json() will convert whatever you send to JSON. As we're sending an object, .send() is enough. But it's not wrong to use .json() as .json() is calling .send() at the end